avm/utilities/pipelines/e2eValidation/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -185,6 +185,43 @@ function Invoke-ResourceRemoval {
             }
             break
         }
+        'Microsoft.Subscription/aliases' {
```

Comment:
As you can see I commented quite a few things on this code snippet. And the reason I did was becauce we're targeting the generic  `Microsoft.Subsciption/aliases' resource type.
While I still think it would be good if the logic below isn't so static & hard-coded, we may want to be more explicit about 'when' this logic is invoked. E.g., instead of just running this snippet for any deployment that happens to use the `'Microsoft.Subscription/aliases'` resource type (which could happen for a future `aliases` module, how about we only run it for your specific pattern.

This would require a few smart adjustments to the case-match though. Instead of just `'Microsoft.Subscription/aliases' {...}` it would need to say something like `{ $PSItem -eq 'Microsoft.Subscription/aliases' -and $ResourceId -like "somethingThatsSpecificToYouLikeTheServiceShortInTheTestFiles" } {...}`, for example.